commands/init.go

```diff
@@ -81,6 +82,10 @@ var initCmd = &cmds.Command{
 func getConfigFromOptions(options cmdkit.OptMap) (*config.Config, error) {
 	newConfig := config.NewDefaultConfig()
 
+	if dir, ok := options[OptionRepoDir].(string); ok {
```

Comment:
Blocking: this goes against the purpose of the recent change.  `sectors` must not be under the repo dir.  It is no one's fault but mine that we are not synced on mental models but we need to converge fast or this is going to bite us.  See #2682 for ways to improve our CLI that keep the sector separate from the repo (either have an OptionSectorsDir that we pass int, or an OptionHomeDir that specifies a home path for initing sector storage and repo)

Reply:
That's a good point. You just did a bunch of work which allows sector stuff to live outside of the repo dir and here I go adding it back.

I've updated the PR to reflect our earlier conversation about initializing with a `--sectordir` flag.